ui/dialog.js

```diff
@@ -412,20 +413,21 @@ return $.widget( "ui.dialog", {
 				},
 				text: false
 			})
-			.addClass( "ui-dialog-titlebar-close" )
 			.appendTo( this.uiDialogTitlebar );
-		this._on( this.uiDialogTitlebarClose, {
-			click: function( event ) {
-				event.preventDefault();
-				this.close( event );
-			}
-		});
 
-		uiDialogTitle = $( "<span>" )
+		this._addClass( this.uiDialogTitlebarClose, "ui-dialog-titlebar-close" )
+			._on( this.uiDialogTitlebarClose, {
+				click: function( event ) {
+					event.preventDefault();
+					this.close( event );
+				}
+			});
+
+		var uiDialogTitle = $( "<span>" )
 			.uniqueId()
-			.addClass( "ui-dialog-title" )
 			.prependTo( this.uiDialogTitlebar );
-		this._title( uiDialogTitle );
+
+		this._addClass( uiDialogTitle, "ui-dialog-title" )._title( uiDialogTitle );
```

Comment:
If we're going to chain these, I'd prefer a multiline statement since this is so uncommon.